hazelcast/src/main/resources/hazelcast-config-3.5.xsd

```diff
@@ -1319,6 +1319,7 @@
     <xs:complexType name="wan-replication-ref">
         <xs:sequence>
```

Comment:
i think, this should be `all` instead of `sequence`,  otherwise order will be important,